data/linux-bridge/0004-bridge-marker-daemonset.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: linux-bridge
+  labels:
+    tier: node
+    app: bridge-marker
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bridge-marker
+    spec:
+      serviceAccountName: bridge-marker
+      hostNetwork: true
```

Comment:
Please test this code on OpenShift. I believe it will fail due to SCC.

Reply:
Yes, the bridge marker user needs the hostnetwork scc.

Reply:
added a bridge-marker scc